pkg/config/output.go

```diff
@@ -579,9 +579,14 @@ func redactStreamKeys(stream *livekit.StreamOutput) {
 }
 
 func clean(filepath string) string {
+	if filepath == "" {
+		return filepath
+	}
+	for strings.HasPrefix(filepath, "../") {
```

Comment:
I think this should be done after the call to `path.Clean`. For instance in this case: `foo/../../bar` will end up as `../bar` after the call to `path.Clean`.